packages/code-du-travail-ui/.eslintrc.yml

```diff
@@ -2,6 +2,6 @@ extends:
   - "@socialgouv/eslint-config-react"
 
 overrides:
-  - files: ["*.test.js"]
+  - files: ["test.js"]
```

Comment:
j'ai l'impression que cette clé ne sert plus

Reply:
la clé `files` ? Ca permet au linter de pas gueuler lorsqu'on édite un fichier de test

Reply:
je la trouve pas dans la doc d'eslint :(

Reply:
https://eslint.org/docs/user-guide/configuring#disabling-rules-only-for-a-group-of-files

Reply:
merci :)